master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Securing access to Calico's metrics endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with prometheus metrics enabled, it may be desirable to use network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
+
+### Purpose of this page
+
+Provide example workflows for creating network policies to limit access to {{site.prodname}}'s metrics endpoints.
+
```

Comment:
Reading further into this page, I am realizing that it includes procedures for different alternative approaches. Since the page is not meant to be followed in a linear fashion, it would be better to include links to the different alternatives here in the introductory section, along with a discussion of each one so that people can make a decision and click to go to the procedure that corresponds to their desired outcome.